acp/acp.go

```diff
@@ -89,10 +89,11 @@ type ACP interface {
 	// Otherwise if check failed then an error is returned (and the boolean result should not be used).
 	//
 	// Note(s):
-	// - permission here is a valid DPI permission we are checking for ("read" or "write").
+	// - permissions here is a valid DPI permission we are checking for ("read" or "write").
+	// - returns true if has access to ANY of the given permissions.
```

Comment:
todo: I think some more documentation of why exactly it behaves like this is important.
Say I call `CheckDocAccess(["delete", "read"])` and it says `true` just because I have "read" access. For a certain action users should have specific permissions (all of them). Not sure how useful it is.

Reply:
- [x] done